qutebrowser/config/config.py

```diff
@@ -363,6 +363,8 @@ class ConfigManager(QObject):
             _get_value_transformer({'false': 'none', 'true': 'debug'}),
         ('ui', 'keyhint-blacklist'):
             _get_value_transformer({'false': '*', 'true': ''}),
+        ('hints', 'auto-follow'):
+            _get_value_transformer({'false': 'never', 'true': 'unique-match'}),
```

Comment:
The problem with `auto-follow = true` is that it's somewhere between - it's like `full-match` for `word`, but `unique-match` for `number`. For the default `letter` mode, both `unique-match` and `full-match` should be the same (see https://github.com/The-Compiler/qutebrowser/pull/1817#issuecomment-238607631), so I don't think the default value matters :wink: In either case, users can choose depending on their mode.